lib/matplotlib/text.py

```diff
@@ -768,7 +794,8 @@ def draw(self, renderer):
             gc.set_url(textobj._url)
             textobj._set_gc_clip(gc)
 
-            angle = textobj.get_rotation()
+            angle = (textobj.get_rotation() + (180.0*transangle/np.pi))%360.0
```

Comment:
Missing spaces around `*`, `/`, and `%`; but in this case, maybe `np.rad2deg` would be clearer?